Feature/terra compose

pixeleet opened the pull request:
Okay, I might have went a bit nuts on this one. 
Wanted to see if I could replace docker-compose with terraform so you wouldn't have to have docker-compose. 

Also I thought it might be nice that we can describe the docker parts in terraform too.

ppvg commented:
Can `variables.tf` be `.gitignore`d?

pixeleet commented:
It can be as long as we instruct the user to create it and/or provide a `variables.example.tf` like with consul config json.

ppvg commented on docker-compose.yml:
```diff
@@ -19,3 +41,8 @@ vault:
+    - "SERVICE_NAME=vault"
+    - "SERVICE_TAGS=vault"
+    - "VAULT_ADDR=http://0.0.0.0:8200"
+    - "VAULT_TOKEN=lofasz"
```
Really?? :confused:

pixeleet replied:
docker-compose.yml needs some love it's a bit outdated. I'll throw a quick
fix in for that.

2015-10-27 10:58 GMT+01:00 Peter-Paul notifications@github.com:

> In docker-compose.yml
> https://github.com/Sweebr/terra-con-vault/pull/1#discussion_r43101276:
> 
> > @@ -19,3 +41,8 @@ vault:
> >      - consul:consul
> >    volumes:
> >      - vault/assets/conf.d:/vault/conf.d
> > -  environment:
> > -    - "SERVICE_NAME=vault"
> > -    - "SERVICE_TAGS=vault"
> > -    - "VAULT_ADDR=http://0.0.0.0:8200"
> > -    - "VAULT_TOKEN=lofasz"
> 
> Really?? [image: :confused:]
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/Sweebr/terra-con-vault/pull/1/files#r43101276.

pixeleet replied:
@ppvg: Updated docker-compose.yml.

2015-10-27 12:40 GMT+01:00 David Lonyai david.lonyai@gmail.com:

> docker-compose.yml needs some love it's a bit outdated. I'll throw a quick
> fix in for that.
> 
> 2015-10-27 10:58 GMT+01:00 Peter-Paul notifications@github.com:
> 
> > In docker-compose.yml
> > https://github.com/Sweebr/terra-con-vault/pull/1#discussion_r43101276:
> > 
> > > @@ -19,3 +41,8 @@ vault:
> > >      - consul:consul
> > >    volumes:
> > >      - vault/assets/conf.d:/vault/conf.d
> > > -  environment:
> > > -    - "SERVICE_NAME=vault"
> > > -    - "SERVICE_TAGS=vault"
> > > -    - "VAULT_ADDR=http://0.0.0.0:8200"
> > > -    - "VAULT_TOKEN=lofasz"
> > 
> > Really?? [image: :confused:]
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > https://github.com/Sweebr/terra-con-vault/pull/1/files#r43101276.

pixeleet commented:
@ppvg Yo ppvg. Introduced a few nice changes.
- consul and nginx config files now can be on any path (by default the examples are copied into where docker expects them to be)
- removed linking where possible

pixeleet commented:
@ppvg this branch is closed use [release/0.1](https://github.com/Sweebr/terra-con-vault/tree/release/0.1) branch and it's corresponding 
[PR](https://github.com/Sweebr/terra-con-vault/pull/2].
